src/datepicker/ngb-date.spec.ts

```diff
@@ -0,0 +1,49 @@
+import {NgbDate} from './ngb-date';
+
+describe('ngb-date', () => {
+
+  it('equals', () => {
+    const date = new NgbDate(2016, 8, 18);
+
+    expect(date.equals(new NgbDate(2016, 8, 18))).toBeTruthy();
+
+    expect(date.equals(null)).toBeFalsy();
+    expect(date.equals(undefined)).toBeFalsy();
```

Comment:
Could you split it into multiple `it` blocks, ex: one for undefined / null, other for "real" dates etc.